disconnect-entitylist.json

```diff
@@ -920,16 +920,6 @@
             "dmtry.com"
         ]
     },
-    "AdOn Network": {
-        "properties": [
-            "adonnetwork.com",
-            "dashboardad.net"
```

Comment:
See the third commit: https://github.com/mozilla-services/shavar-prod-lists/pull/19/commits/b8105c1cd09e5f8d2f8e54882351e6d77c78aa7a